### ISSUE TITLE
`lora.Linear.bias` should point to `lora.Linear.base_layer.bias` as the `lora.Linear.weight` does

### ISSUE THREAD

**Issue by ArthurZucker** [huggingface/peft#1524]
### System Info

- `transformers` version: 4.39.0.dev0
- Platform: Linux-5.4.0-166-generic-x86_64-with-glibc2.31
- Python version: 3.9.17
- Huggingface_hub version: 0.21.1
- Safetensors version: 0.4.2
- Accelerate version: 0.27.2
- Accelerate config:    not found
- PyTorch version (GPU?): 2.2.0+cu121 (True)
- Tensorflow version (GPU?): 2.13.0 (True)
- Flax version (CPU?/GPU?/TPU?): 0.7.0 (cpu)
- Jax version: 0.4.13
- JaxLib version: 0.4.13
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>
- Peft version: 0.9.0
- Trl version: 0.7.11

### Who can help?

@pacman100 @BenjaminBossan 

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder
- [ ] My own task or dataset (give details below)

### Reproduction

I am running the following: 

```python
from datasets import load_dataset
from trl import SFTTrainer
from peft import LoraConfig
from transformers import AutoTokenizer, AutoModelForCausalLM, TrainingArguments
import torch

model_id = "ArthurZ/mamba-130m"
tokenizer = AutoTokenizer.from_pretrained(model_id, pad_token ="<|padding|>")
model = AutoModelForCausalLM.from_pretrained(model_id, pad_token_id = 1, torch_dtype = torch.float32, residual_in_fp32 =True, device_map = {"":1})

dataset = load_dataset("Abirate/english_quotes", split="train")


training_args = TrainingArguments(
    output_dir="./results",
    num_train_epochs=5,
    per_device_train_batch_size=4,
    logging_dir='./logs',
    logging_steps=10,
    learning_rate=1e-3
)
lora_config =  LoraConfig(
        r=8,
        target_modules=["x_proj", "in_proj", "embeddings"],
        task_type="CAUSAL_LM",
        bias="none"
)
lora_config = None
trainer = SFTTrainer(
    model=model,
    tokenizer=tokenizer,
    args=training_args,
    peft_config=lora_config,
    train_dataset=dataset,
    dataset_text_field="quote",
)

trainer.train()
```

from https://github.com/huggingface/transformers/pull/28094 (Add mamba).
TLDR I want to run a custom kernel:
```python
            contextualized_states = mamba_inner_fn(
                projected_states,
                self.conv1d.weight,
                self.conv1d.bias if self.use_conv_bias else None,
                self.x_proj.weight,
                self.dt_proj.weight,
                self.out_proj.weight,
                self.out_proj.bias.float() if self.use_bias else None,
                -torch.exp(self.A_log.float()),
                None,  # input-dependent B
                None,  # input-dependent C
                self.D.float(),
                delta_bias=self.dt_proj.bias.float(),
                delta_softplus=True,
            )
```
but `self.dt_proj.bias` no longer exists. My understanding is that `self.dt_proj` is now a `lora_linear` and `self.dt_proj.weight` is accessible and works fine. Why not the bias? 

### Expected behavior

When you access the `weight`  it points to the `base_layer.weight`, when you want to access the `bias` you have an error. If you are not training on the bias you should still be able to manually add the bias.

**Comment by BenjaminBossan** [comment]
@ArthurZucker I created a PR: #1530. `bias` being a "read only" attribute (via `@property`) should be enough, right?
